example_dot_env

```diff
@@ -1,3 +0,0 @@
-SESSION_SECRET='1845f5419299977a57a61e74c659ad804dca6f7fd5209b3888e773f123b113f8'
-
-SUNLIGHT_FOUNDATION_KEY='55a27bdc46b947c4b63b791b7cf6fa2f'
```

Comment:
To clarify my thinking on including these in the repo, these are dummy keys not used in production.  It's meant as a template for someone who wants to get their own app up and running quickly.